README.md

```diff
@@ -85,7 +115,7 @@ The `message` action retrieves a response to a user's input. The parameters that
 
 * `output` : System output. Include the output from the request when you have several requests within the same Dialog turn to pass back in the intermediate information.
 
-If you haven't created a package binding, you can invoke the message action by providing all the required parameters, i.e.:
+If you haven't created a service binding, you can invoke the message action by providing all the required parameters, i.e.:
```

Comment:
don't use `i.e.` because people don't know what it means. Start a new sentence and use `For example:`